app/stylesheets/pages/quizzes/_quizzes.scss

```diff
@@ -532,7 +532,6 @@ li.quiz {
 .question_bank {
   border-bottom: 1px solid #b6babf;
   border-top: 1px solid #fff;
-  overflow: hidden;
```

Comment:
The engineer that looked at it suspected this as the reason it's not working in Chrome.